src/vireshJJun20/Assignment24/ValidatePhNumber.java

```diff
@@ -0,0 +1,75 @@
+package vireshJJun20.Assignment24;
+
+public class ValidatePhNumber {
+
+	boolean flag=true;
+
+	boolean isNumCorrect(String num) {
```

Comment:
Instead of num we can take something like phoneNumber/phnNum etc because it will help us identify that we are actually working on a phone number and not just a number. 

Also please reduce the number of line spacing within the program

Reply:
Changes made, please check. Thanks.